packages/devtools_app/lib/src/inspector/inspector_screen.dart

```diff
@@ -93,6 +94,46 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
     blockWhileInProgress(inspectorController.collapseDetailsToSelected);
   }
 
+  Widget _buildErrorList(
+    BuildContext context,
+    List<InspectableWidgetError> errors,
+  ) {
+    return ListView(
+      padding: const EdgeInsets.all(8),
+      children: <Widget>[
+        for (final error in errors)
+          Padding(
+            padding: const EdgeInsets.only(bottom: 8),
+            // TODO(dantup): How to make the cursor a hand pointer?
+            child: GestureDetector(
+              onTap: () => inspectorController.updateSelectionFromService(
+                  firstFrame: false, inspectorRef: error.inspectorRef),
+              child: Row(
+                children: [
+                  Padding(
+                    padding: const EdgeInsets.only(right: 8),
+                    child: Container(
+                      padding: const EdgeInsets.symmetric(horizontal: 3.0),
+                      decoration: BoxDecoration(
+                        color: const Color.fromARGB(0xff, 0xF4, 0x43, 0x36),
```

Comment:
nit: move all colors to constants next to color colors from our "theme"

Reply:
This code was removed, and now uses a shared existing color.